src/main/kotlin/com/terraformation/backend/accelerator/api/ModulesController.kt

```diff
@@ -42,47 +34,28 @@ import org.springframework.web.multipart.MultipartFile
 @RequestMapping("/api/v1/accelerator/modules")
 @RestController
 class ModulesController(
```

Comment:
Updated modules to return only modules data. Moved cohort modules information to under cohorts controller.